src/Sign.SignatureProviders.TrustedSigning/TrustedSigningService.cs

```diff
@@ -69,12 +67,13 @@ public async Task<X509Certificate2> GetCertificateAsync(CancellationToken cancel
 
                     _logger.LogTrace(Resources.FetchingCertificate);
 
-                    CertificateProfileSignOperation operation = await _client.StartSignAsync(_accountName, _certificateProfileName, _emptyRequest, cancellationToken: cancellationToken);
-                    Response<SignStatus> response = await operation.WaitForCompletionAsync(cancellationToken);
+                    var response = await _client.GetSignCertificateChainAsync(_accountName, _certificateProfileName, cancellationToken: cancellationToken);
```

Comment:
This project doesn't use `var` when the type is not directly visible on the right side.

Reply:
Shouldn't that be covered by the editorconfig? I saw no feedback. I updated the PR to use the explicit type. Thanks!